sdcm/sct_events.py

```diff
@@ -160,6 +160,30 @@ class SystemEvent(SctEvent):
     pass
 
 
+class TestFrameworkEvent(SctEvent):  # pylint: disable=too-many-instance-attributes
+    def __init__(self, source, source_method,  # pylint: disable=redefined-builtin,too-many-arguments
+                 exception=None, message=None, args=None, kwargs=None, severity=None):
+        super().__init__()
+        if severity is None:
+            self.severity = Severity.CRITICAL
+        else:
+            self.severity = severity
+        self.source = source
+        self.source_method = source_method
+        self.exception = exception
+        self.message = message
+        self.args = args
```

Comment:
I don't see any code using `messgae` or `args` or `kwargs`, why it's part of this PR ?

Reply:
because SetUp does not have any, it is reserved for future use

Reply:
I think you could easily add those once needed, extending apis in python is that hard...